File: OsmAnd-java/src/main/java/net/osmand/data/QuadPoint.java
package net.osmand.data;

public class QuadPoint {
	public float x;
	public float y;

	public QuadPoint() {
	}

	public QuadPoint(float x, float y) {
		this.x = x;
		this.y = y;
	}

	public QuadPoint(QuadPoint a) {
		this(a.x, a.y);
	}

	public void set(float x, float y) {
		this.x = x;
		this.y = y;
	}

	@Override
	public String toString() {
		return "x " + x + " y " + y;
	}
}

File: OsmAnd-shared/src/commonMain/kotlin/net/osmand/shared/data/QuadPoint.kt
package net.osmand.shared.data

class QuadPoint {
	var x: Float = 0.0f
	var y: Float = 0.0f

	constructor()

	constructor(x: Float, y: Float) {
		this.x = x
		this.y = y
	}

	constructor(a: QuadPoint) : this(a.x, a.y)

	fun set(x: Float, y: Float) {
		this.x = x
		this.y = y
	}

	override fun toString(): String {
		return "x $x y $y"
	}
}
